lib/services/samples-service.ts

```diff
@@ -122,28 +126,46 @@ export class SamplesService implements ISamplesService {
 					return;
 				}
 
-				outputLines.push(util.format("%s:%s======================", category.name, os.EOL));
+				outputLines.push(util.format("   %s:%s   ======================", category.name, os.EOL));
 
 				_.each(category.samples, (sample: Sample) => {
-					var nameRow = util.format("    Sample: %s", sample.displayName);
-					var descriptionRow = util.format("    Description: %s", sample.description);
-					var gitClone = util.format("    Github repository page: %s", sample.githubUrl);
-					var cloneCommand = util.format("    Clone command: $ appbuilder sample clone %s", sample.name);
+					var nameRow = util.format("      Sample: %s", sample.displayName);
+					var descriptionRow = util.format("      Description: %s", sample.description);
+					var gitClone = util.format("      Github repository page: %s", sample.githubUrl);
+					var cloneCommand = util.format("      Clone command: $ appbuilder sample clone %s", sample.name);
 					outputLines.push([nameRow, descriptionRow, gitClone, cloneCommand].join(os.EOL));
 				});
 			});
 
-			outputLines.unshift("You can choose a sample from the following:");
 			return outputLines.join(os.EOL + os.EOL);
 		}).future<string>()();
 	}
 
-	private getAllSamples(): IFuture<Sample[]> {
-		return (() => {
-			var iceniumOrganizationSamples = this.getRepositories(SamplesService.GITHUB_ICENIUM_LOCATION_ENDPOINT, (repo: any) => SamplesService.GITHUB_REGEX.test(repo.clone_url) && !repo[SamplesService.REMOTE_LOCK_STATE_PRIVATE]).wait();
-			var telerikOrganizationSamples = this.getRepositories(SamplesService.GITHUB_TELERIK_LOCATION_ENDPOINT, (repo: any) => repo.name === SamplesService.NATIVESCRIPT_SAMPLE_CUTENESS_NAME).wait();
-			var repos = _.union(iceniumOrganizationSamples, telerikOrganizationSamples);
+	private getRegExpForFramework(framework?: string): RegExp {
+		var regex: RegExp;
+		framework = framework || "";
+		switch(framework.toLowerCase()) {
+			case this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.NativeScript.toLowerCase():
+				regex = SamplesService.GITHUB_NS_SAMPLES_REGEX;
```

Comment:
:facepalm: (I really need this emoticon for this case) 
I'll fix it, thanks for noticing this.